code/game/objects/buckling.dm

```diff
@@ -58,31 +58,67 @@
 	add_fingerprint(user)
 	unbuckle_mob()
 
-	if(buckle_mob(M))
-		if(M == user)
-			M.visible_message(\
-				SPAN_NOTICE("[M.name] buckles themselves to [src]."),\
-				SPAN_NOTICE("You buckle yourself to [src]."),\
-				SPAN_NOTICE("You hear metal clanking."))
-		else
-			M.visible_message(\
-				SPAN_DANGER("[M.name] is buckled to [src] by [user.name]!"),\
-				SPAN_DANGER("You are buckled to [src] by [user.name]!"),\
-				SPAN_NOTICE("You hear metal clanking."))
+	
+	
+	var/buckleTime = 25
+	if (M == user)
+		user.set_dir(src.dir)
+		buckleTime = 8
+		user.visible_message(\
+		SPAN_NOTICE("[user] attempts to buckle themselves into \the [src]!</span>"),\
```

Comment:
This message is unnecessary noise, someone trying to buckle themselves is not important for everyone to see.

Reply:
deleted this